index.html

```diff
@@ -1,56 +1,188 @@
-            complicated thoughts with clarity.
-			
-        </main>
-
-        <aside>
-            <h2>Aside Goes Here</h2><br>Quintilian declaring Cicero was "not the name of a man, but of eloquence
-            itself." He is credited with transforming Latin from a modest utilitarian language into a versatile literary
-            medium capable of expressing abstract and complicated
-        </aside>
-
-        <footer>
-            <h2>Footer Goes Here</h2>During the chaotic latter half of the first century BC, marked by civil wars and
-            the dictatorship of Gaius Julius Caesar, Cicero championed a return to the traditional republican
-            government.
-            <p>Thanks to Dr. William Helling of IUPUI and w3schools.com (https://www.w3schools.com/css/css_grid.asp) -- Kasper Valtakari</p>
-        </footer>
-
-    </div><!-- END GRID CONTAINER  -->
-
-</body>
-
-</html>
\ No newline at end of file
+		<meta
+			name="description"
+			content="Helping developers make their sites accessible for all."
+		/>
+
+		<meta
+			name="keywords"
+			content="inclusive, inclusive design, design, colorblind, screen reader, accessible, accessibility"
+		/>
+
+		<!-- Fonts: ROBOTO 300, 400, 500, 700 -->
+		<link rel="preconnect" href="https://fonts.googleapis.com" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+		<link
+			href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap"
+			rel="stylesheet"
+		/>
+
+		<!-- Favicon -->
+		<link rel="icon" href="images/eye.svg" />
+
+		<!-- CSS -->
+		<link rel="stylesheet" href="styles/main.css" />
+
+		<!-- Title -->
+		<title>Design for Everyone: Overview</title>
+	</head>
+	<body>
+		<!-- Grid-container -->
+		<div class="grid-container">
+			<!-- Header -->
+			<div class="sticky">
+				<!-- ^this container sticks the header and nav to the side-->
+				<header class="header">
+					<a href="index.html" class="homeButton"
+						><img src="images/logo-placeholder.png" alt="logo/home button"
+					/></a>
+				</header>
+
+				<!-- Nav -->
+				<nav class="nav">
+					<ul class="navList">
+						<li class="current">
+							<a class="navLink" href="#">Overview</a>
+						</li>
+
+						<li class="navItem">
+							<a class="navLink" href="pages/text.html">Text and Readability</a>
+						</li>
+
+						<li class="navItem">
+							<a class="navLink" href="pages/semantics.html">Semantics</a>
+						</li>
+
+						<li class="navItem">
+							<a class="navLink" href="pages/rwd.html">Responsive Design</a>
+						</li>
+
+						<li class="navItem">
+							<a class="navLink" href="pages/file-size.html">File Size</a>
+						</li>
+
+						<li class="navItem">
+							<a class="navLink" href="pages/organization.html">Organization</a>
+						</li>
+					</ul>
+				</nav>
+			</div>
+
+			<!-- Main -->
+			<main class="main">
+				<!-- Top Image -->
+				<img

```

Comment:
Nit: With the changes from the wireframes to the high fidelity designs, this top image area feels more like a `<header>` than the home button imo.